include/libp2p/multi/content_identifier_codec.hpp

```diff
@@ -42,16 +43,23 @@ namespace libp2p::multi {
         gsl::span<const uint8_t> bytes);
 
     /**
-     * @brief Encode CID v0 to string representation
+     * @brief Encode CID to string representation
      * @param cid - input CID for encode
      * @param encoding - type of the encoding, ignored for CID v0 (always
      * Base58)
-     * @todo Sergey Kaprovich: FIL-133 add support for CID v1
      * @return CID string
      */
     static outcome::result<std::string> toString(
         const ContentIdentifier &cid,
         MultibaseCodec::Encoding encoding = MultibaseCodec::Encoding::BASE58);
```

Comment:
Change default to BASE32_LOWER.
https://github.com/ipfs/go-cid/blob/master/cid.go#L347

Reply:
Both CID versions (0 and 1) can be encoded with Base58, and if multibase codec version is not explicitly specified - we assume, that this is CID v0, which can be encoded only with Base58. 
We should change default to BASE32_LOWER here if we plan to use it only with CIDv1

Reply:
Maybe we can add "auto" enum value, or separate method that accepts "base" param, and let "toString()" use default BASE32_LOWER.